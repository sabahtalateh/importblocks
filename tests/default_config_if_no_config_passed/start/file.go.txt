package test1

import(
	"github.com/a/b"
	"os"
	"x/y/z"
	"github.com/sabahtalateh/importblocks/b"
	"fmt"
	"math"
	"github.com/sabahtalateh/importblocks/c"
	"github.com/sabahtalateh/importblocks/a"
	"bytes"
)
