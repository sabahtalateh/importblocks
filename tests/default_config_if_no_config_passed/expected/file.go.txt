package test1

import (
	"bytes"
	"fmt"
	"math"
	"os"

	"github.com/a/b"
	"x/y/z"

	"github.com/sabahtalateh/importblocks/a"
	"github.com/sabahtalateh/importblocks/b"
	"github.com/sabahtalateh/importblocks/c"
)
